.github/workflows/integrations-test.yml

```diff
@@ -21,7 +21,7 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Install poetry
-        run: pipx install poetry
+        run: pipx install 'poetry>=1.0.0,<2.0.0'
 
```

Comment:
why its not just uses dev requirements of ocean?

Reply:
Because we install it manually both here and in the makefile